.gitignore

```diff
@@ -19,6 +19,7 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+venv
```

Comment:
put this down with 'env/' and add the slash.

Reply:
done